### DISCUSSION TITLE
Walktime :: Not Found

### DISCUSSION THREAD

**Discussion by rehanguha** [anuraghazra/github-readme-stats#1846] [Q&A] [answered]
https://github-readme-stats.vercel.app/api/wakatime?username=rehanguha

Something went wrong! file an issue at https://tiny.one/readme-stats
Wakatime user not found, make sure you have a wakatime profile

**Comment by rickstaa** [comment, marked as answer]
@rehanguha Thanks for your question. The error message indicates that your Wakatime username does not exist (see https://wakatime.com/@rehanguha). Maybe your profile is not public, or you made a typo when generating the Wakatime username?

```md
[![willianrod's wakatime stats](https://github-readme-stats.vercel.app/api/wakatime?username=rehanguha)](https://github.com/anuraghazra/github-readme-stats)
```

[![willianrod's wakatime stats](https://github-readme-stats.vercel.app/api/wakatime?username=rehanguha)](https://github.com/anuraghazra/github-readme-stats)

Hope that helps!

**Reply by rickstaa** [reply]
@rehanguha Did that answer your question?

**Reply by rehanguha** [reply]
Yup... Sorry to keep the issue open.. Yes it did answer the question. Thank you very much.

**Reply by rickstaa** [reply]
Great to hear! Could you maybe accept my answer to that we can mark this question as answered 🤓?
